install/kubernetes/cilium/values.yaml

```diff
@@ -2020,3 +2020,7 @@ enableK8sTerminatingEndpoint: true
 
 # -- Configure whether to unload DNS policy rules on graceful shutdown
 # dnsPolicyUnloadOnShutdown: false
+
+# -- Configure the key of the taint indicating that Cilium is not ready on the node.
+# When set to a value starting with `ignore-taint.cluster-autoscaler.kubernetes.io/`, the Cluster Autoscaler will ignore the taint on it's decisions, allowing the cluster to scale up.
```

Comment:
```suggestion
# When set to a value starting with `ignore-taint.cluster-autoscaler.kubernetes.io/`, the Cluster Autoscaler will ignore the taint on its decisions, allowing the cluster to scale up.
```
And the line below, last of the file, doesn't end with a newline (this is probably a matter on how your editor is configured). Same thing for install/kubernetes/cilium/templates/cilium-configmap.yaml.

Reply:
Added final newline. Thanks for the review. @qmonnet